independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/model/ServerResourceMethod.java

```diff
@@ -79,4 +80,12 @@ public String getActualDeclaringClassName() {
     public void setActualDeclaringClassName(String actualDeclaringClassName) {
         this.actualDeclaringClassName = actualDeclaringClassName;
     }
+
+    public String getClassOfMetWithJaxRsEndpointDefAnn() {
```

Comment:
I can't say I understand what this is supposed to do from the name, so can we please add some Javadoc?

Reply:
I'll add Javadoc describing this is a class declaring amethod that has `@Path` or method like `@GET` and no path. This is previous behavior. The name is not good.

Reply:
> The name is not good

I know all about coming up with bad names myself :)